src/Corrade/Utility/Sha1.h

```diff
@@ -31,6 +31,7 @@
 
 #include <string>
 
+#include "Corrade/Containers/ArrayView.h"
```

Comment:
Could be just the Containers.h header here and ArrayView.h included in the cpp.

Reply:
It's now needed for the

~~~
        /** @overload */
        template<std::size_t size>
        Sha1& operator<<(const char(&data)[size]) {
            return *this << Containers::arrayView(data, size);
        }
~~~